### ISSUE TITLE
Modal no working

### ISSUE THREAD

**Issue by jmdmao** [ssuperczynski/ngx-easy-table#108]
Even following the orientation inserted in the template  

> turn clickEvent in the configuration.service.ts to true to get row events 
  and display whatever you want here

 changing the `clickEvent` from `false` to` true`, the modal does not appear.

In live edit, all other modifications on `configuration.service.ts`, such as `selectRow` or `paginationEnabled`, works, but `clickEvent` does not affect anything.

**Comment by ssuperczynski** [comment]
Could you check demo now: https://stackblitz.com/edit/ngx-easy-table?file=app%2Fcomponents%2Fmodal%2Fmodal.component.html

I fixed it couple hours ago.

**Comment by jmdmao** [comment]
I tested here and these were the results:

![image](https://user-images.githubusercontent.com/29717874/39247913-b73a8f7a-4868-11e8-95bc-c21f4f0cc31e.png)

![image](https://user-images.githubusercontent.com/29717874/39247595-d49c89f2-4867-11e8-8cd5-40f2d982262a.png)

**Comment by jmdmao** [comment]
In my personal project, I'm temporarily using the ngx-bootstrap modal as a substitute:

![image](https://user-images.githubusercontent.com/29717874/39248568-743209e0-486a-11e8-832d-f0c2d8645c24.png)

**Comment by ssuperczynski** [comment]
@jmdmao what version of ngx-easy-table you have in the package.json?

**Comment by jmdmao** [comment]
`"ngx-easy-table": "^2.4.1",`

do you wanna see the whole package.json?

**Comment by jmdmao** [comment]
Sorry for taking the opportunity, but how could I change the search placeholder text (my project is in Brazilian Portuguese)? Or any other customization variable, such as disabling hover?

**Comment by ssuperczynski** [comment]
So far table doesn't have option to change this text. Will be fixed in the new version.

Now you can change it by:

```ts
  ngAfterViewInit() {
    const phonePlaceholeder = document.getElementById('search_phone');
    phonePlaceholeder.placeholder = 'new value';
  }
```

**Comment by jmdmao** [comment]
thanks!

**Comment by ssuperczynski** [comment]
@jmdmao placeholder fixed in version 3.2.
Demo -> https://ngx-easy-table.stackblitz.io/column-width :)
